.github/workflows/create-release-branch.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
 
     - name: Bump latest tag variable version
       if: ${{ !github.event.inputs.version }}
-      run: echo "RELEASE_VERSION=$(echo ${{ env.CURRENT_VERSION }} | sed -r 's/([0-9]+)\.([0-9]+)\.([0-9]+)/echo \1.$((\2+1)).\3/' | sh)" >> $GITHUB_ENV
+      run: echo "RELEASE_VERSION=$(echo ${{ env.CURRENT_VERSION }} | sed -r 's/([0-9]+)\.([0-9]+)\.([0-9]+)/echo \1.$((\2+1)).0/' | sh)" >> $GITHUB_ENV
```

Comment:
Just a thought a simple `awk` script might be a better choice here e.g. something like this should do the job
```bash
cho "RELEASE_VERSION=$(echo ${{ env.CURRENT_VERSION }} | awk -F. -v OFS=. '{$2 += 1 ; $3 = 0; print}')" >> $GITHUB_ENV
```

Reply:
@tb06904 if this sounds good to you, I think for now I will leave this as it isn't crucial to solve the issue and this regex is present on the other repos. Worth maybe thinking about improving though when we come to refactor the release pipeline generally.

Reply:
I would echo this. There are at least as many instances of this `sed` command as there are Gaffer repos, probably a couple more. A change like this should be done as a refactor to keep consistency.

Reply:
That's fair doe we have a ticket for the refactor?

Reply:
Not yet, we should probably make one though. I think the main points I would like to do are:
- removing the `ADMIN_GITHUB_TOKEN`
- removing release branches